views/group/index.pug

```diff
@@ -9,9 +9,9 @@ block content
   if paginationOpt.pageNum > 1
     include ./pagination
 
-  .cards
+  .columns.is-desktop.is-multiline.is-centered
     each group in groups
-      .card.mb-2.mt-4
+      .column.is-5-desktop.card.mx-4.my-4
```

Comment:
```suggestion
      .column.is-5-desktop.card.mx-4.my-4.flex-column
```

Have seen this UI problem:
![Screenshot 2020-09-08 16 24 53](https://user-images.githubusercontent.com/60111772/92490266-19c56280-f1f1-11ea-93f4-210b5d024f0b.png)

Implementing into `additions.scss` and using `.flex-column` on line `.column.is-5 ...`:
```css
.flex-column {
  display: flex;
  flex-direction: column;

  >.card-footer {
    margin-top: auto;
  }
}
```

It looked like this on web (didn't break on mobile view):
![Screenshot 2020-09-08 16 26 42](https://user-images.githubusercontent.com/60111772/92490409-4b3e2e00-f1f1-11ea-9d41-ac24654723b0.png)

Reply:
Good catch, implemented it